plugin/pkg/scheduler/scheduler.go

```diff
@@ -270,16 +270,21 @@ func (sched *Scheduler) scheduleOne() {
 
 	// Tell the cache to assume that a pod now is running on a given node, even though it hasn't been bound yet.
 	// This allows us to keep scheduling without waiting on binding to occur.
-	assumedPod := *pod
+	podCopy, err := api.Scheme.DeepCopy(pod)
```

Comment:
Could you add a comment to why you are doing this, it's totally non-obvious.

Reply:
Done.